controllers/app_controller.js

```diff
@@ -40,5 +41,16 @@ const getCommentsByReview_id = (req, res, next) => {
     .catch(next)
 }
 
+const postCommentWithReview_id = (req, res, next) => {
+    const review_id = req.params.review_id
+    const { body, username } = req.body
+    fetchReviewById(review_id).then(() => {
```

Comment:
Why do we need to fetch the review first, is it because what happens if we try and add to a review that doesnt exist, I think in this case psql would error for a foreign key violation and you could use a psql error handler